src/build-config/webpack.js

```diff
@@ -0,0 +1,48 @@
+export const registerSvgWebpackLoader = (config) =>
+{
+	// Grab the existing rule that handles SVG imports
+	const fileLoaderRule = config.module.rules.find((rule) => rule.test?.test?.(".svg") && rule.resourceQuery?.toString() !== "/raw/");
+	// Modify the file loader rule to ignore *.svg, since we have it handled now.
+	fileLoaderRule.exclude = /\.svg$/i;
```

Comment:
We should

- [ ] add this `exclude` to every rule, not just the first we find (there might be multiple ones)
- [ ] should merge the `exclude` with a maybe existing `exclude`
- [ ] find out why we tested for `/raw/` above